src/app.js

```diff
@@ -239,6 +239,13 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   const pluginLoader = appManager.pluginLoader
   const workspace = pluginLoader.get()
   const engine = new Engine(appManager)
+  engine.setPluginOption = (profile) => {
```

Comment:
I think it's ok to do it like that, but if you need to overide more stuff you might want to extends the Engine at some point.